generators/network/templates/is_running.sh

```diff
@@ -6,7 +6,7 @@
 #
 # Exit on first error, print all commands.
 set +ev
-NUM_CONTAINERS=$(docker ps -f label=fabric-environment-name="<%= name %>" -q | wc -l | tr -d ' ')
+NUM_CONTAINERS=$(docker container port $(docker ps -f label=fabric-environment-name="<%= name %>" -q) | grep <%-port%>/tcp | wc -l | tr -d ' ')
```

Comment:
Why are we looking at ports and environment name instead of just environment name? (same for windows)

Reply:
I believe I hit a problem during development (can't remember exactly what caused it), so I changed it to be this so it was more specific.